raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -222,20 +223,26 @@ def get_pending_transfer_pairs(
     return pending_pairs
 
 
+def _fee_for_channel(channel: NettingChannelState, amount: PaymentAmount) -> FeeAmount:
+    balance = get_balance(channel.our_state, channel.partner_state)
```

Comment:
I think what we should use here is the channel capacity. I had the capacity and the balance switched in my mind, so I might have said something wrong. See https://github.com/raiden-network/raiden/pull/4319

Reply:
There are arguments for both capacity and balance.

Pro capacity:
* Most payments succeed, so this is usually the more likely/more up to date state

Pro balance:
* This is the only really confirmed state
* There is no way to influence the fees by starting payments to change the capacity of the mediator. If someone is crazy enough to invest the effort, they could first make a payment to move the mediator's capacity in one direction, then send a second payment to reap the decreased fees. He could make the first payment fail and still keep the reduced fees for the second payment.

I currently favor the balance as the safer bet.